vms/rpcchainvm/vm_server.go

```diff
@@ -591,10 +582,7 @@ func (vm *VMServer) CrossChainAppResponse(ctx context.Context, msg *vmpb.CrossCh
 }
 
 func (vm *VMServer) AppRequest(ctx context.Context, req *vmpb.AppRequestMsg) (*emptypb.Empty, error) {
-	nodeID, err := ids.ToNodeID(req.NodeId)
-	if err != nil {
-		return nil, err
-	}
+	nodeID := ids.NodeIDFromBytes(req.NodeId, len(req.NodeId))
```

Comment:
This should error if the bytes aren't 32 or 20 😄

Reply:
Done